solr/core/src/java/org/apache/solr/servlet/SolrDispatchFilter.java

```diff
@@ -196,7 +196,7 @@ public void init(FilterConfig config) throws ServletException
 
     }finally{
       log.trace("SolrDispatchFilter.init() done");
-      this.cores = coresInit; // crucially final assignment 
+      this.cores = coresInit; // crucially final assignment
```

Comment:
```suggestion
      this.cores = coresInit; // crucial final assignment
```
What makes this crucial?

Reply:
they're new lines which added. otherwise it's same as upstream